lib/logging/third_party_transaction.rb

```diff
@@ -44,7 +45,7 @@ def wrap_with_logging(*method_names, additional_class_logs: {}, additional_insta
     # these will be included after instance instantiation, making them available
     # to the instance and retaining their scope.
     module ScopedInstanceMethods
-      def log_3pi_begin(method_name, additional_class_logs = {}, additional_instance_logs = {}, args = [])
```

Comment:
These default args are redundant, as the calling method already has defaults set.